PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+| Q                 | A
+| ----------------- | ----------
+| Bug fix?          | yes/no
+| New feature?      | yes/no    <!-- please update the /CHANGELOG.md file -->
+| BC breaks?        | no     
+| Related Issue     | Fix #...  <!-- will close issue automatically, if any -->
+| Need Doc update   | yes/no
+
+
+## Description
```

Comment:
I like splitting up in `What is changed?` / `Why did you do this?` ( / `How can we test this?`)

Reply:
Yeah, good idea! "Descrption" is probably to general 👌